### PR TITLE
Security updates are added in couple of events in edition-agen-1 version

### PR DESCRIPTION
**Pull request by zrdgthr**
### Description of the Change
Security updates are added in couple of events in edition-agen-1 version

### CODE REVIEW

File: src/test/resources/input/ActivityStarted.json [hunk 1]
```diff
@@ -13,6 +13,9 @@
         "name": "name",
         "uri": "http:\/\/java.sun.com\/j2se\/1.3\/",
         "serializer":"pkg:maven"
```
Comment (xvinosi-github) [inline src/test/resources/input/ActivityStarted.json]: Correct the values in input

File: src/test/resources/input/ActivityTriggered.json [hunk 2]
```diff
@@ -14,10 +14,9 @@
         "uri": "http:\/\/java.sun.com\/j2se\/1.3\/",
         "serializer":"pkg:maven"
```
Comment (xvinosi-github) [inline src/test/resources/input/ActivityTriggered.json]: Same as above

File: src/test/resources/input/ActivityTriggered.json [hunk 3]
```diff
@@ -2,21 +2,17 @@
...
+        "domainId": "eiffeltest",
+        "host": "localhost",
+        "name": "LOCALHOST",
+        "uri": "https://localhost:8080/jenkins/job",
+        "serializer":"pkg:maven/com.github.eiffel-community/eiffel-remrem-semantics@2.0.2"
       },
-      "sdm": {
-          "authorIdentity": "test",
-          "encryptedDigest": "sample"
+      "secutiry":{
```
Comment (xvinosi-github) [inline src/test/resources/input/ActivityTriggered.json]: security typo errror.

File: src/test/resources/input/ActivityStarted.json [hunk 4]
```diff
@@ -2,49 +2,49 @@
...
-      "executionUri": "http:\/\/myHost.com\/firstLog",
+      "executionUri": "http://localhost:8080/job/dsd",
       "liveLogs": [{
-          "name": "ericsson",
-          "uri": "http:\/\/myHost.com\/firstLog"
+          "name": "console",
+          "uri": "http://localhost:8085/job/dsd/9/console"
         },
         {
           "name": "eric",
```
Comment (xvinosi-github) [inline src/test/resources/input/ActivityStarted.json]: remove the eric object, only console should do.

File: src/main/java/com/ericsson/eiffel/semantics/events/EiffelActivityTriggeredEvent.java [hunk 5]
```diff
@@ -19,7 +19,9 @@
 import com.google.gson.annotations.Expose;
 import com.google.gson.annotations.SerializedName;
 
-public class EiffelActivityTriggeredEvent extends Event {
+public class EiffelActivityTriggeredEvent
```
Comment (xvinosi-github) [inline src/main/java/com/ericsson/eiffel/semantics/events/EiffelActivityTriggeredEvent.java]: formatting

File: src/main/java/com/ericsson/eiffel/semantics/events/EiffelActivityStartedEventMeta.java [hunk 6]
```diff
@@ -169,7 +170,7 @@ public void setSecurity(Security security) {
         private final static Map<String, EiffelActivityStartedEventMeta.Type> CONSTANTS = new HashMap<String, EiffelActivityStartedEventMeta.Type>();
 
         static {
-            for (EiffelActivityStartedEventMeta.Type c : values()) {
+            for (EiffelActivityStartedEventMeta.Type c: values()) {
```
Comment (xvinosi-github) [inline src/main/java/com/ericsson/eiffel/semantics/events/EiffelActivityStartedEventMeta.java]: formatting.

File: src/main/java/com/ericsson/eiffel/semantics/events/EiffelActivityStartedEvent.java [hunk 7]
```diff
@@ -21,83 +21,83 @@
...
-    private EiffelActivityStartedEventData data;
-    /**
-     * 
-     * (Required)
-     * 
-     */
-    @SerializedName("links")
-    @Expose
-    private List<Link> links = new ArrayList<Link>();
+	/**
```
Comment (SantoshNC68) [inline src/main/java/com/ericsson/eiffel/semantics/events/EiffelActivityStartedEvent.java]: remove the tabs and add 4 spaces

File: src/main/java/com/ericsson/eiffel/semantics/events/EiffelActivityStartedEventData.java [hunk 8]
```diff
@@ -55,4 +55,5 @@ public void setCustomData(List<CustomData> customData) {
         this.customData = customData;
     }
 
+
```
Comment (SantoshNC68) [inline src/main/java/com/ericsson/eiffel/semantics/events/EiffelActivityStartedEventData.java]: remove this line

File: src/main/java/com/ericsson/eiffel/semantics/events/EiffelActivityStartedEvent.java (hunk 7 above)
Comment (SantoshNC68) [reply src/main/java/com/ericsson/eiffel/semantics/events/EiffelActivityStartedEvent.java]: keep the old file if no changes.

File: src/main/java/com/ericsson/eiffel/semantics/events/EiffelActivityStartedEventMeta.java [hunk 9]
```diff
@@ -21,7 +21,8 @@
 import com.google.gson.annotations.Expose;
 import com.google.gson.annotations.SerializedName;
 
-public class EiffelActivityStartedEventMeta implements Meta {
+public class EiffelActivityStartedEventMeta implements Meta
+{
```
Comment (SantoshNC68) [inline src/main/java/com/ericsson/eiffel/semantics/events/EiffelActivityStartedEventMeta.java]: keep the old format

File: src/main/java/com/ericsson/eiffel/semantics/events/EiffelActivityTriggeredEvent.java (hunk 5 above)
Comment (SantoshNC68) [reply src/main/java/com/ericsson/eiffel/semantics/events/EiffelActivityTriggeredEvent.java]: keep the old format.

File: src/main/java/com/ericsson/eiffel/semantics/events/EiffelActivityStartedEventMeta.java (hunk 6 above)
Comment (xvinosi-github) [reply src/main/java/com/ericsson/eiffel/semantics/events/EiffelActivityStartedEventMeta.java]: Still not formatted?

File: src/main/java/com/ericsson/eiffel/semantics/events/EiffelActivityTriggeredEventData.java [hunk 10]
```diff
@@ -97,15 +97,18 @@ public void setCustomData(List<CustomData> customData) {
...
+        @SerializedName("AUTOMATED")
+        AUTOMATED("AUTOMATED"),
+        @SerializedName("OTHER")
         OTHER("OTHER");
         private final String value;
         private final static Map<String, EiffelActivityTriggeredEventData.ExecutionType> CONSTANTS = new HashMap<String, EiffelActivityTriggeredEventData.ExecutionType>();
 
         static {
-            for (EiffelActivityTriggeredEventData.ExecutionType c : values()) {
+            for (EiffelActivityTriggeredEventData.ExecutionType c: values()) {
```
Comment (xvinosi-github) [inline src/main/java/com/ericsson/eiffel/semantics/events/EiffelActivityTriggeredEventData.java]: not formatted?

**Pull request merged**